### DISCUSSION TITLE
Coding Stellarium with Cython

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#2880] [General]
I have pyCharm professional IDE for python programs. It has also cython, which is python to C portion. Is it possible for example code with python bookmarks features to Stellarium with pyCharm and port it to C / C++ with cython?

**Comment by alex-w** [comment]
We recommend QtCreator + we prepared settings for it

**Reply by ilkant** [reply]
That is not freeware and costs a lot...

**Reply by 10110111** [reply]
How come QtCreator is not freeware? It's free and opensource, just as Qt is.

**Reply by alex-w** [reply]
Downloads for open source users is always free

**Comment by gzotti** [comment]
There is a bookmarks feature written in the usual C++. There is still some work to do there, though.  In C++.

But it may be interesting to find out whether it is possible to write a Python plugin (skeleton) for Stellarium, or some other tutorial-like inter-language efforts.
